planpro_importer/reader.py

```diff
@@ -1,15 +1,18 @@
-from yaramo.model import Topology, Node, Signal, Edge
+from yaramo.model import Topology, Node, Signal, Edge, DbrefGeoNode
 from planpro_importer import model
 
 
 class PlanProReader(object):
 
-    def __init__(self, plan_pro_file_name):
+    def __init__(self, plan_pro_file_name, x_coordinate_shift=4533770.0, y_coordinate_shift=5625780.0):
```

Comment:
is this supposed to be the shift between WGS84 and Dbref? if yes, I would suggest using the existing functionality from https://github.com/simulate-digital-rail/yaramo/blob/main/yaramo/geo_point.py#L59-L62. Otherwise, it would be great to have a short explanation for this feature or an encapsulation in a new subclass of GeoPoint.

Reply:
Sorry, this full pull request makes no sense. I will refactor it.

Reply:
@jeriox Removed, please review again.